schema.sql

```diff
@@ -24,4 +24,15 @@ ALTER TABLE animals DROP COLUMN species;
 
```

Comment:
![image](https://github.com/ezraermy/vet_clinic/assets/93039359/69e64a2d-8169-42e7-8b6c-36bfab0a4750)

- [ ] As suggested by the previous reviewer, kindly add all the screenshots in the pull request summary to follow the project requirements 👍